app/views/RoomInfoEditView/index.tsx

```diff
@@ -523,15 +523,13 @@ class RoomInfoEditView extends React.Component<IRoomInfoEditViewProps, IRoomInfo
 			<KeyboardView
 				style={{ backgroundColor: themes[theme].surfaceRoom }}
 				contentContainerStyle={sharedStyles.container}
-				keyboardVerticalOffset={128}
-			>
+				keyboardVerticalOffset={128}>
```

Comment:
Can you remove this file from this PR?
Either you commited unintentionally, or Prettier might be messing up with you